2020-03-04-all-hands-meeting.md

```diff
@@ -0,0 +1,194 @@
+        - pc: excited. trust guardian project. bl: heard good things about witness project. Starling project wants to tell a use story for dweb tech. all: no blocks.
+    - Fin: [Public holiday policy](https://github.com/hyphacoop/organizing/issues/156#issuecomment-591363595)
+        - Rough wording:
+            - >The Co-operative recommends Members to not work on a public holiday, but in the case where Members voluntarily chooses to or under special circumstances, they will receive regular wages and "substitute time off", and there should be no expectation of "public holiday plus premium pay".
+                >
+                >Since Members have variable hours of work per period, and a 5% annualized public holiday pay is already included in Member wages, Members are already entitled to take substitute time off as needed, and will be paid regular wage for that work, as compliant with the [Employment Standards Act of Ontario](https://www.ontario.ca/document/your-guide-employment-standards-act-0/public-holidays).
+
+- vote on all: **CONSENSUS** by all to accept above decisions.
+
+- [Social media role: selection](https://hackmd.io/vWuT-UQUQquwKnqU3812dQ?view) (5 min)
+    - pc walks through social media role
+    - pc: permission to diverge from existing format for role definitions?
+    - uv: what does _role_ mean? Always existing?
+        - pc: a defined person doing this
+        - uv: like Cultivator?
+            - pc: yes
+        - uv: if no one volunteers to do it, what happens?
+        - uv: perhaps we should just vote to let this be in bizdev domain?
+            - all: +1
+
+- Decision making and client notes privacy (5 min)
+    - Is a decision like having a project codename an org-level or project-level decision?
+    - Would a codename protect identity of a client enough for careful discussion in non-private notes?
+    - uv: are the codenames just for obfuscation? is there another reason?
+        - pc: didn't get into that part...
+    - bl: this isn't a convo wrt how project team wants to communicate, it's how we collectively decided to relate to our clients
+    - yj: hard line to balance... public vs. private vs. not-public
+    - uv: very strongly pre-sales cycle needs to be secret, bc we don't have permission from orgs involved...
+    - uv: once things become a project, we can discuss with client, but we _need_ permission from client
+    - uv: "codename use within project" vs. "codename as privacy" are two different conversations
+    - uv: codename is a project-level decision, but making a client's internal info public is an org-level decision (ben: +1)
+    - **DECISION**: from now on, all business development convos (e.g. future clients) should happen in the _private_ room of `#hyphacoop-business-planning`. All are welcome to use the space, not just WG members.
+        - bl adjusted the room topic to reflect
+
+- Walk-throughs and **decide next steps** (15 min, stretch) _skipped_
```

Comment:
fyi added to this week's pad

Reply:
Probably need to wait for next Full bc today is dc's

Reply:
yes.